addons/arsenal/functions/fnc_addSort.sqf

```diff
@@ -37,8 +37,7 @@ params [
     ["_class", "", [""]],
     ["_title", "", [""]],
     ["_statement", {}, [{}]],
-    ["_condition", {true}, [{}]],
-    ["_overwrite", false, [false]]
```

Comment:
Was `_overwrite` just unused?

Reply:
> Was `_overwrite` just unused?

Yea.
I wanted to add overwriting functionalit in #9040, but never did (and I don't think I ever will). I forgot to remove it though before committing.